Makefile

```diff
@@ -420,31 +420,31 @@ talosctl-image: ## Builds the talosctl container image and outputs it to the reg
 	@$(MAKE) registry-talosctl
 
 talosctl-all:
-	@$(MAKE) local-talosctl-all DEST=$(ARTIFACTS) PUSH=false NAME=Client
```

Comment:
I believe this was never actually used, but this causes very fun side-effects when running integration tests locally